src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -21,6 +28,12 @@ public void actionPerformed(ActionEvent e) {
 		{
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 		}
+		else if (e.getActionCommand().toLowerCase().equals("rectangle factory")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.